qa/tasks/scripts/ses_tuned.sh

```diff
@@ -0,0 +1,30 @@
+set -ex
+
+function check_tuned() {
+    role=$1
+    shift
+    minions="$@" # this renders rest of arguments as single space separated list
```

Comment:
sorry about that, we need to use 'local' keywords for local function variables, otherwise it can override the global ones.